### ISSUE TITLE
Key Authorization file doesn't match challenge. Using Pebble for testing.

### ISSUE THREAD

**Issue by wlmarion** [acmesh-official/acme.sh#4602]
I am having an issue where key authorization is failing. I am using Pebble for testing. I get the following: Verify error:The key authorization file from the server did not match this challenge. I can see the token exchange in the debug logs, but am unable to determine why it isn't getting matched up properly.

Steps to reproduce
------------------

 acme.sh --issue -d example.com --webroot /var/www/html --server https://10.6.18.10:14000/dir --insecure --apache --debug 2

Debug log
-----------------

 acme.sh --issue -d example.com --webroot /var/www/html --server https://10.6.18.10:14000/dir --insecure --apache --debug 2
[Mon Apr 17 08:23:16 PM UTC 2023] _selectServer try snames='zerossl.com,zerossl'
[Mon Apr 17 08:23:16 PM UTC 2023] _selectServer try snames='letsencrypt.org,letsencrypt'
[Mon Apr 17 08:23:16 PM UTC 2023] _selectServer try snames='letsencrypt.org_test,letsencrypt_test,letsencrypttest'
[Mon Apr 17 08:23:16 PM UTC 2023] _selectServer try snames='buypass.com,buypass'
[Mon Apr 17 08:23:16 PM UTC 2023] _selectServer try snames='buypass.com_test,buypass_test,buypasstest'
[Mon Apr 17 08:23:17 PM UTC 2023] _selectServer try snames='ssl.com,sslcom'
[Mon Apr 17 08:23:17 PM UTC 2023] _selectServer try snames='google.com,google'
[Mon Apr 17 08:23:17 PM UTC 2023] _selectServer try snames='google.com_test,googletest,google_test'
[Mon Apr 17 08:23:17 PM UTC 2023] Lets find script dir.
[Mon Apr 17 08:23:17 PM UTC 2023] _SCRIPT_='/root/.acme.sh/acme.sh'
[Mon Apr 17 08:23:17 PM UTC 2023] _script='/root/.acme.sh/acme.sh'
[Mon Apr 17 08:23:17 PM UTC 2023] _script_home='/root/.acme.sh'
[Mon Apr 17 08:23:17 PM UTC 2023] Using config home:/root/.acme.sh
[Mon Apr 17 08:23:17 PM UTC 2023] LE_WORKING_DIR='/root/.acme.sh'
https://github.com/acmesh-official/acme.sh
v3.0.6
[Mon Apr 17 08:23:17 PM UTC 2023] Using server: https://10.6.18.10:14000/dir
[Mon Apr 17 08:23:17 PM UTC 2023] Running cmd: issue
[Mon Apr 17 08:23:17 PM UTC 2023] _main_domain='example.com'
[Mon Apr 17 08:23:17 PM UTC 2023] _alt_domains='no'
[Mon Apr 17 08:23:17 PM UTC 2023] Using config home:/root/.acme.sh
[Mon Apr 17 08:23:17 PM UTC 2023] ACME_DIRECTORY='https://10.6.18.10:14000/dir'
[Mon Apr 17 08:23:17 PM UTC 2023] _ACME_SERVER_HOST='10.6.18.10'
[Mon Apr 17 08:23:17 PM UTC 2023] _ACME_SERVER_PATH='dir'
[Mon Apr 17 08:23:17 PM UTC 2023] DOMAIN_PATH='/root/.acme.sh/example.com_ecc'
[Mon Apr 17 08:23:17 PM UTC 2023] '/var/www/html,apache' does not contain 'dns'
[Mon Apr 17 08:23:17 PM UTC 2023] Le_NextRenewTime
[Mon Apr 17 08:23:17 PM UTC 2023] Using ACME_DIRECTORY: https://10.6.18.10:14000/dir
[Mon Apr 17 08:23:17 PM UTC 2023] _init api for server: https://10.6.18.10:14000/dir
[Mon Apr 17 08:23:17 PM UTC 2023] GET
[Mon Apr 17 08:23:17 PM UTC 2023] url='https://10.6.18.10:14000/dir'
[Mon Apr 17 08:23:17 PM UTC 2023] timeout=
[Mon Apr 17 08:23:17 PM UTC 2023] _CURL='curl --silent --dump-header /root/.acme.sh/http.header  -L  --trace-ascii /tmp/tmp.dYpdMLQFNy  -g  --insecure  '
[Mon Apr 17 08:23:17 PM UTC 2023] ret='0'
[Mon Apr 17 08:23:17 PM UTC 2023] response='{
   "keyChange": "https://10.6.18.10:14000/rollover-account-key",
   "meta": {
      "externalAccountRequired": false,
      "termsOfService": "data:text/plain,Do%20what%20thou%20wilt"
   },
   "newAccount": "https://10.6.18.10:14000/sign-me-up",
   "newNonce": "https://10.6.18.10:14000/nonce-plz",
   "newOrder": "https://10.6.18.10:14000/order-plz",
   "revokeCert": "https://10.6.18.10:14000/revoke-cert"
}'
[Mon Apr 17 08:23:17 PM UTC 2023] ACME_KEY_CHANGE='https://10.6.18.10:14000/rollover-account-key'
[Mon Apr 17 08:23:17 PM UTC 2023] ACME_NEW_AUTHZ
[Mon Apr 17 08:23:17 PM UTC 2023] ACME_NEW_ORDER='https://10.6.18.10:14000/order-plz'
[Mon Apr 17 08:23:17 PM UTC 2023] ACME_NEW_ACCOUNT='https://10.6.18.10:14000/sign-me-up'
[Mon Apr 17 08:23:17 PM UTC 2023] ACME_REVOKE_CERT='https://10.6.18.10:14000/revoke-cert'
[Mon Apr 17 08:23:17 PM UTC 2023] ACME_AGREEMENT='data:text/plain,Do%20what%20thou%20wilt'
[Mon Apr 17 08:23:17 PM UTC 2023] ACME_NEW_NONCE='https://10.6.18.10:14000/nonce-plz'
[Mon Apr 17 08:23:18 PM UTC 2023] Using CA: https://10.6.18.10:14000/dir
[Mon Apr 17 08:23:18 PM UTC 2023] _on_before_issue
[Mon Apr 17 08:23:18 PM UTC 2023] _chk_main_domain='example.com'
[Mon Apr 17 08:23:18 PM UTC 2023] _chk_alt_domains
[Mon Apr 17 08:23:18 PM UTC 2023] '/var/www/html,apache' does not contain 'no'
[Mon Apr 17 08:23:18 PM UTC 2023] Le_LocalAddress
[Mon Apr 17 08:23:18 PM UTC 2023] d='example.com'
[Mon Apr 17 08:23:18 PM UTC 2023] Check for domain='example.com'
[Mon Apr 17 08:23:18 PM UTC 2023] _currentRoot='/var/www/html'
[Mon Apr 17 08:23:18 PM UTC 2023] d
[Mon Apr 17 08:23:18 PM UTC 2023] '/var/www/html,apache' contains 'apache'
[Mon Apr 17 08:23:18 PM UTC 2023] Using config home:/root/.acme.sh
[Mon Apr 17 08:23:18 PM UTC 2023] ACME_DIRECTORY='https://10.6.18.10:14000/dir'
[Mon Apr 17 08:23:18 PM UTC 2023] _ACME_SERVER_HOST='10.6.18.10'
[Mon Apr 17 08:23:18 PM UTC 2023] _ACME_SERVER_PATH='dir'
[Mon Apr 17 08:23:18 PM UTC 2023] httpdconfname='apache2.conf'
[Mon Apr 17 08:23:18 PM UTC 2023] httpdroot='/etc/apache2'
[Mon Apr 17 08:23:18 PM UTC 2023] httpdconf='/etc/apache2/apache2.conf'
[Mon Apr 17 08:23:18 PM UTC 2023] httpdconfname='apache2.conf'
[Mon Apr 17 08:23:18 PM UTC 2023] Checking if there is an error in the apache config file before starting.
[Mon Apr 17 08:23:18 PM UTC 2023] OK
[Mon Apr 17 08:23:18 PM UTC 2023] Backup apache config file='/etc/apache2/apache2.conf'
[Mon Apr 17 08:23:18 PM UTC 2023] JFYI, Config file /etc/apache2/apache2.conf is backuped to /root/.acme.sh/apache2.conf
[Mon Apr 17 08:23:18 PM UTC 2023] In case there is an error that can not be restored automatically, you may try restore it yourself.
[Mon Apr 17 08:23:18 PM UTC 2023] The backup file will be deleted on success, just forget it.
[Mon Apr 17 08:23:18 PM UTC 2023] apacheVer='2.4.52'
[Mon Apr 17 08:23:19 PM UTC 2023] _saved_account_key_hash='CNliP4tpdPM9Q4Maqgtfpo4z1emsrVz3sbB8sAJPcqE='
[Mon Apr 17 08:23:19 PM UTC 2023] _saved_account_key_hash is not changed, skip register account.
[Mon Apr 17 08:23:19 PM UTC 2023] Read key length:ec-256
[Mon Apr 17 08:23:19 PM UTC 2023] _createcsr
[Mon Apr 17 08:23:19 PM UTC 2023] domain='example.com'
[Mon Apr 17 08:23:19 PM UTC 2023] domainlist
[Mon Apr 17 08:23:19 PM UTC 2023] csrkey='/root/.acme.sh/example.com_ecc/example.com.key'
[Mon Apr 17 08:23:19 PM UTC 2023] csr='/root/.acme.sh/example.com_ecc/example.com.csr'
[Mon Apr 17 08:23:19 PM UTC 2023] csrconf='/root/.acme.sh/example.com_ecc/example.com.csr.conf'
[Mon Apr 17 08:23:19 PM UTC 2023] Single domain='example.com'
[Mon Apr 17 08:23:19 PM UTC 2023] seg='example'
[Mon Apr 17 08:23:19 PM UTC 2023] _is_idn_d='example.com'
[Mon Apr 17 08:23:19 PM UTC 2023] _idn_temp
[Mon Apr 17 08:23:19 PM UTC 2023] _is_idn_d='example.com'
[Mon Apr 17 08:23:19 PM UTC 2023] _idn_temp
[Mon Apr 17 08:23:19 PM UTC 2023] _csr_cn='example.com'
[Mon Apr 17 08:23:19 PM UTC 2023] seg='example'
[Mon Apr 17 08:23:19 PM UTC 2023] Getting domain auth token for each domain
[Mon Apr 17 08:23:19 PM UTC 2023] seg='example'
[Mon Apr 17 08:23:19 PM UTC 2023] _is_idn_d='example.com'
[Mon Apr 17 08:23:19 PM UTC 2023] _idn_temp
[Mon Apr 17 08:23:19 PM UTC 2023] d
[Mon Apr 17 08:23:19 PM UTC 2023] _identifiers='{"type":"dns","value":"example.com"}'
[Mon Apr 17 08:23:19 PM UTC 2023] _notBefore
[Mon Apr 17 08:23:19 PM UTC 2023] _notAfter
[Mon Apr 17 08:23:19 PM UTC 2023] url='https://10.6.18.10:14000/order-plz'
[Mon Apr 17 08:23:19 PM UTC 2023] payload='{"identifiers": [{"type":"dns","value":"example.com"}]}'
[Mon Apr 17 08:23:19 PM UTC 2023] EC key
[Mon Apr 17 08:23:19 PM UTC 2023] Get nonce with HEAD. ACME_NEW_NONCE='https://10.6.18.10:14000/nonce-plz'
[Mon Apr 17 08:23:19 PM UTC 2023] HEAD
[Mon Apr 17 08:23:19 PM UTC 2023] _post_url='https://10.6.18.10:14000/nonce-plz'
[Mon Apr 17 08:23:19 PM UTC 2023] body
[Mon Apr 17 08:23:19 PM UTC 2023] _postContentType='application/jose+json'
[Mon Apr 17 08:23:20 PM UTC 2023] _CURL='curl --silent --dump-header /root/.acme.sh/http.header  -L  --trace-ascii /tmp/tmp.42Wa6lwfxs  -g  --insecure   -I  '
[Mon Apr 17 08:23:20 PM UTC 2023] _ret='0'
[Mon Apr 17 08:23:20 PM UTC 2023] _headers='HTTP/2 200
cache-control: public, max-age=0, no-cache
link: <https://10.6.18.10:14000/dir>;rel="index"
replay-nonce: CxRcqVhO5O_Es3PryDQ3mA
date: Mon, 17 Apr 2023 20:23:20 GMT
'
[Mon Apr 17 08:23:20 PM UTC 2023] _CACHED_NONCE='CxRcqVhO5O_Es3PryDQ3mA'
[Mon Apr 17 08:23:20 PM UTC 2023] nonce='CxRcqVhO5O_Es3PryDQ3mA'
[Mon Apr 17 08:23:20 PM UTC 2023] POST
[Mon Apr 17 08:23:20 PM UTC 2023] _post_url='https://10.6.18.10:14000/order-plz'
[Mon Apr 17 08:23:20 PM UTC 2023] body='{"protected": "eyJub25jZSI6ICJDeFJjcVZoTzVPX0VzM1ByeURRM21BIiwgInVybCI6ICJodHRwczovLzEwLjYuMTguMTA6MTQwMDAvb3JkZXItcGx6IiwgImFsZyI6ICJFUzI1NiIsICJraWQiOiAiaHR0cHM6Ly8xMC42LjE4LjEwOjE0MDAwL215LWFjY291bnQvNGZhYTQ3OTU4MGM1Y2Y3YyJ9", "payload": "eyJpZGVudGlmaWVycyI6IFt7InR5cGUiOiJkbnMiLCJ2YWx1ZSI6ImV4YW1wbGUuY29tIn1dfQ", "signature": "97-fapFCIh7oQeLhO7CkPitkjMqlr2qYPX2GJZq2OSnU-dvLb7maoI8txVspZo0q0b7TXVK4qkIpUStPYqmzaQ"}'
[Mon Apr 17 08:23:20 PM UTC 2023] _postContentType='application/jose+json'
[Mon Apr 17 08:23:20 PM UTC 2023] Http already initialized.
[Mon Apr 17 08:23:20 PM UTC 2023] _CURL='curl --silent --dump-header /root/.acme.sh/http.header  -L  --trace-ascii /tmp/tmp.42Wa6lwfxs  -g  --insecure  '
[Mon Apr 17 08:23:20 PM UTC 2023] _ret='0'
[Mon Apr 17 08:23:20 PM UTC 2023] responseHeaders='HTTP/2 201
cache-control: public, max-age=0, no-cache
content-type: application/json; charset=utf-8
link: <https://10.6.18.10:14000/dir>;rel="index"
location: https://10.6.18.10:14000/my-order/NS4kCB8cv9pSCgbOBCNIKKTLitL2LpZKlYsWyYu3sgA
replay-nonce: 6XYCyMGmFfqsnLfGSvXpRQ
content-length: 376
date: Mon, 17 Apr 2023 20:23:20 GMT
'
[Mon Apr 17 08:23:20 PM UTC 2023] code='201'
[Mon Apr 17 08:23:20 PM UTC 2023] original='{
   "status": "pending",
   "expires": "2023-04-18T20:23:20Z",
   "identifiers": [
      {
         "type": "dns",
         "value": "example.com"
      }
   ],
   "finalize": "https://10.6.18.10:14000/finalize-order/NS4kCB8cv9pSCgbOBCNIKKTLitL2LpZKlYsWyYu3sgA",
   "authorizations": [
      "https://10.6.18.10:14000/authZ/9fGSeiH6Vf5QsZo_L9mOV_nMWyDGYEpxRPu3S0HanWA"
   ]
}'
[Mon Apr 17 08:23:20 PM UTC 2023] response='{"status":"pending","expires":"2023-04-18T20:23:20Z","identifiers":[{"type":"dns","value":"example.com"}],"finalize":"https://10.6.18.10:14000/finalize-order/NS4kCB8cv9pSCgbOBCNIKKTLitL2LpZKlYsWyYu3sgA","authorizations":["https://10.6.18.10:14000/authZ/9fGSeiH6Vf5QsZo_L9mOV_nMWyDGYEpxRPu3S0HanWA"]}'
[Mon Apr 17 08:23:20 PM UTC 2023] Le_LinkOrder='https://10.6.18.10:14000/my-order/NS4kCB8cv9pSCgbOBCNIKKTLitL2LpZKlYsWyYu3sgA'
[Mon Apr 17 08:23:20 PM UTC 2023] Le_OrderFinalize='https://10.6.18.10:14000/finalize-order/NS4kCB8cv9pSCgbOBCNIKKTLitL2LpZKlYsWyYu3sgA'
[Mon Apr 17 08:23:20 PM UTC 2023] _authorizations_seg='https://10.6.18.10:14000/authZ/9fGSeiH6Vf5QsZo_L9mOV_nMWyDGYEpxRPu3S0HanWA'
[Mon Apr 17 08:23:20 PM UTC 2023] _authz_url='https://10.6.18.10:14000/authZ/9fGSeiH6Vf5QsZo_L9mOV_nMWyDGYEpxRPu3S0HanWA'
[Mon Apr 17 08:23:20 PM UTC 2023] url='https://10.6.18.10:14000/authZ/9fGSeiH6Vf5QsZo_L9mOV_nMWyDGYEpxRPu3S0HanWA'
[Mon Apr 17 08:23:20 PM UTC 2023] payload
[Mon Apr 17 08:23:20 PM UTC 2023] Use cached jwk for file: /root/.acme.sh/ca/10.6.18.10/dir/account.key
[Mon Apr 17 08:23:20 PM UTC 2023] Use _CACHED_NONCE='6XYCyMGmFfqsnLfGSvXpRQ'
[Mon Apr 17 08:23:20 PM UTC 2023] nonce='6XYCyMGmFfqsnLfGSvXpRQ'
[Mon Apr 17 08:23:20 PM UTC 2023] POST
[Mon Apr 17 08:23:20 PM UTC 2023] _post_url='https://10.6.18.10:14000/authZ/9fGSeiH6Vf5QsZo_L9mOV_nMWyDGYEpxRPu3S0HanWA'
[Mon Apr 17 08:23:20 PM UTC 2023] body='{"protected": "eyJub25jZSI6ICI2WFlDeU1HbUZmcXNuTGZHU3ZYcFJRIiwgInVybCI6ICJodHRwczovLzEwLjYuMTguMTA6MTQwMDAvYXV0aFovOWZHU2VpSDZWZjVRc1pvX0w5bU9WX25NV3lER1lFcHhSUHUzUzBIYW5XQSIsICJhbGciOiAiRVMyNTYiLCAia2lkIjogImh0dHBzOi8vMTAuNi4xOC4xMDoxNDAwMC9teS1hY2NvdW50LzRmYWE0Nzk1ODBjNWNmN2MifQ", "payload": "", "signature": "tJLgeyU8gWqp-9L3etV2L0sCemxTwy02s3071vwYONXZd7215svPXdNjqvU_UqXoCzM-OXHs5gad6I236knXTA"}'
[Mon Apr 17 08:23:20 PM UTC 2023] _postContentType='application/jose+json'
[Mon Apr 17 08:23:20 PM UTC 2023] Http already initialized.
[Mon Apr 17 08:23:21 PM UTC 2023] _CURL='curl --silent --dump-header /root/.acme.sh/http.header  -L  --trace-ascii /tmp/tmp.42Wa6lwfxs  -g  --insecure  '
[Mon Apr 17 08:23:21 PM UTC 2023] _ret='0'
[Mon Apr 17 08:23:21 PM UTC 2023] responseHeaders='HTTP/2 200
cache-control: public, max-age=0, no-cache
content-type: application/json; charset=utf-8
link: <https://10.6.18.10:14000/dir>;rel="index"
replay-nonce: puZ61M9QiAw_wGIld80Y3Q
content-length: 865
date: Mon, 17 Apr 2023 20:23:21 GMT
'
[Mon Apr 17 08:23:21 PM UTC 2023] code='200'
[Mon Apr 17 08:23:21 PM UTC 2023] original='{
   "status": "pending",
   "identifier": {
      "type": "dns",
      "value": "example.com"
   },
   "challenges": [
      {
         "type": "dns-01",
         "url": "https://10.6.18.10:14000/chalZ/xb1u810-X6xhw1qR0GWxmVCEQW3HRnvbOoRhjNmjr1I",
         "token": "fnF-rz6rDf9ViF31KNfGQ20EtmGxSK6TiKGkOQYCarI",
         "status": "pending"
      },
      {
         "type": "tls-alpn-01",
         "url": "https://10.6.18.10:14000/chalZ/LYbsRUZ9joM0y_TQ6xDA0VkWWyUZ3QPiCGi6wVsmCAY",
         "token": "Yf60kzVyuixpWfRPdjo58a5OAOnPwJtq_NrvwGASjeM",
         "status": "pending"
      },
      {
         "type": "http-01",
         "url": "https://10.6.18.10:14000/chalZ/iMsAmXHhZWscfLkCCDoCH9OyhdIeNOt3IWULgVHp4OE",
         "token": "ilNH3x8ybrSE8UFBxpevq72CtMhD13FEKA08bq8zo4w",
         "status": "pending"
      }
   ],
   "expires": "2023-04-17T21:23:20Z"
}'
[Mon Apr 17 08:23:21 PM UTC 2023] response='{"status":"pending","identifier":{"type":"dns","value":"example.com"},"challenges":[{"type":"dns-01","url":"https://10.6.18.10:14000/chalZ/xb1u810-X6xhw1qR0GWxmVCEQW3HRnvbOoRhjNmjr1I","token":"fnF-rz6rDf9ViF31KNfGQ20EtmGxSK6TiKGkOQYCarI","status":"pending"},{"type":"tls-alpn-01","url":"https://10.6.18.10:14000/chalZ/LYbsRUZ9joM0y_TQ6xDA0VkWWyUZ3QPiCGi6wVsmCAY","token":"Yf60kzVyuixpWfRPdjo58a5OAOnPwJtq_NrvwGASjeM","status":"pending"},{"type":"http-01","url":"https://10.6.18.10:14000/chalZ/iMsAmXHhZWscfLkCCDoCH9OyhdIeNOt3IWULgVHp4OE","token":"ilNH3x8ybrSE8UFBxpevq72CtMhD13FEKA08bq8zo4w","status":"pending"}],"expires":"2023-04-17T21:23:20Z"}'
[Mon Apr 17 08:23:21 PM UTC 2023] response='{"status":"pending","identifier":{"type":"dns","value":"example.com"},"challenges":[{"type":"dns-01","url":"https://10.6.18.10:14000/chalZ/xb1u810-X6xhw1qR0GWxmVCEQW3HRnvbOoRhjNmjr1I","token":"fnF-rz6rDf9ViF31KNfGQ20EtmGxSK6TiKGkOQYCarI","status":"pending"},{"type":"tls-alpn-01","url":"https://10.6.18.10:14000/chalZ/LYbsRUZ9joM0y_TQ6xDA0VkWWyUZ3QPiCGi6wVsmCAY","token":"Yf60kzVyuixpWfRPdjo58a5OAOnPwJtq_NrvwGASjeM","status":"pending"},{"type":"http-01","url":"https://10.6.18.10:14000/chalZ/iMsAmXHhZWscfLkCCDoCH9OyhdIeNOt3IWULgVHp4OE","token":"ilNH3x8ybrSE8UFBxpevq72CtMhD13FEKA08bq8zo4w","status":"pending"}],"expires":"2023-04-17T21:23:20Z"}'
[Mon Apr 17 08:23:21 PM UTC 2023] _d='example.com'
[Mon Apr 17 08:23:21 PM UTC 2023] _authorizations_map='example.com,{"status":"pending","identifier":{"type":"dns","value":"example.com"},"challenges":[{"type":"dns-01","url":"https://10.6.18.10:14000/chalZ/xb1u810-X6xhw1qR0GWxmVCEQW3HRnvbOoRhjNmjr1I","token":"fnF-rz6rDf9ViF31KNfGQ20EtmGxSK6TiKGkOQYCarI","status":"pending"},{"type":"tls-alpn-01","url":"https://10.6.18.10:14000/chalZ/LYbsRUZ9joM0y_TQ6xDA0VkWWyUZ3QPiCGi6wVsmCAY","token":"Yf60kzVyuixpWfRPdjo58a5OAOnPwJtq_NrvwGASjeM","status":"pending"},{"type":"http-01","url":"https://10.6.18.10:14000/chalZ/iMsAmXHhZWscfLkCCDoCH9OyhdIeNOt3IWULgVHp4OE","token":"ilNH3x8ybrSE8UFBxpevq72CtMhD13FEKA08bq8zo4w","status":"pending"}],"expires":"2023-04-17T21:23:20Z"}
'
[Mon Apr 17 08:23:21 PM UTC 2023] d='example.com'
[Mon Apr 17 08:23:21 PM UTC 2023] Getting webroot for domain='example.com'
[Mon Apr 17 08:23:21 PM UTC 2023] _w='/var/www/html'
[Mon Apr 17 08:23:21 PM UTC 2023] _currentRoot='/var/www/html'
[Mon Apr 17 08:23:21 PM UTC 2023] _is_idn_d='example.com'
[Mon Apr 17 08:23:21 PM UTC 2023] _idn_temp
[Mon Apr 17 08:23:21 PM UTC 2023] _candidates='example.com,{"status":"pending","identifier":{"type":"dns","value":"example.com"},"challenges":[{"type":"dns-01","url":"https://10.6.18.10:14000/chalZ/xb1u810-X6xhw1qR0GWxmVCEQW3HRnvbOoRhjNmjr1I","token":"fnF-rz6rDf9ViF31KNfGQ20EtmGxSK6TiKGkOQYCarI","status":"pending"},{"type":"tls-alpn-01","url":"https://10.6.18.10:14000/chalZ/LYbsRUZ9joM0y_TQ6xDA0VkWWyUZ3QPiCGi6wVsmCAY","token":"Yf60kzVyuixpWfRPdjo58a5OAOnPwJtq_NrvwGASjeM","status":"pending"},{"type":"http-01","url":"https://10.6.18.10:14000/chalZ/iMsAmXHhZWscfLkCCDoCH9OyhdIeNOt3IWULgVHp4OE","token":"ilNH3x8ybrSE8UFBxpevq72CtMhD13FEKA08bq8zo4w","status":"pending"}],"expires":"2023-04-17T21:23:20Z"}'
[Mon Apr 17 08:23:21 PM UTC 2023] response='{"status":"pending","identifier":{"type":"dns","value":"example.com"},"challenges":[{"type":"dns-01","url":"https://10.6.18.10:14000/chalZ/xb1u810-X6xhw1qR0GWxmVCEQW3HRnvbOoRhjNmjr1I","token":"fnF-rz6rDf9ViF31KNfGQ20EtmGxSK6TiKGkOQYCarI","status":"pending"},{"type":"tls-alpn-01","url":"https://10.6.18.10:14000/chalZ/LYbsRUZ9joM0y_TQ6xDA0VkWWyUZ3QPiCGi6wVsmCAY","token":"Yf60kzVyuixpWfRPdjo58a5OAOnPwJtq_NrvwGASjeM","status":"pending"},{"type":"http-01","url":"https://10.6.18.10:14000/chalZ/iMsAmXHhZWscfLkCCDoCH9OyhdIeNOt3IWULgVHp4OE","token":"ilNH3x8ybrSE8UFBxpevq72CtMhD13FEKA08bq8zo4w","status":"pending"}],"expires":"2023-04-17T21:23:20Z"}'
[Mon Apr 17 08:23:21 PM UTC 2023] entry='"type":"http-01","url":"https://10.6.18.10:14000/chalZ/iMsAmXHhZWscfLkCCDoCH9OyhdIeNOt3IWULgVHp4OE","token":"ilNH3x8ybrSE8UFBxpevq72CtMhD13FEKA08bq8zo4w","status":"pending"'
[Mon Apr 17 08:23:21 PM UTC 2023] token='ilNH3x8ybrSE8UFBxpevq72CtMhD13FEKA08bq8zo4w'
[Mon Apr 17 08:23:21 PM UTC 2023] uri='https://10.6.18.10:14000/chalZ/iMsAmXHhZWscfLkCCDoCH9OyhdIeNOt3IWULgVHp4OE'
[Mon Apr 17 08:23:21 PM UTC 2023] keyauthorization='ilNH3x8ybrSE8UFBxpevq72CtMhD13FEKA08bq8zo4w.FW1KGPj1HSHl9wu_d4wcIsBAj1CkQErlXT6vxu847yo'
[Mon Apr 17 08:23:21 PM UTC 2023] dvlist='example.com#ilNH3x8ybrSE8UFBxpevq72CtMhD13FEKA08bq8zo4w.FW1KGPj1HSHl9wu_d4wcIsBAj1CkQErlXT6vxu847yo#https://10.6.18.10:14000/chalZ/iMsAmXHhZWscfLkCCDoCH9OyhdIeNOt3IWULgVHp4OE#http-01#/var/www/html'
[Mon Apr 17 08:23:21 PM UTC 2023] d
[Mon Apr 17 08:23:21 PM UTC 2023] vlist='example.com#ilNH3x8ybrSE8UFBxpevq72CtMhD13FEKA08bq8zo4w.FW1KGPj1HSHl9wu_d4wcIsBAj1CkQErlXT6vxu847yo#https://10.6.18.10:14000/chalZ/iMsAmXHhZWscfLkCCDoCH9OyhdIeNOt3IWULgVHp4OE#http-01#/var/www/html,'
[Mon Apr 17 08:23:21 PM UTC 2023] d='example.com'
[Mon Apr 17 08:23:21 PM UTC 2023] ok, let's start to verify
[Mon Apr 17 08:23:21 PM UTC 2023] Verifying: example.com
[Mon Apr 17 08:23:21 PM UTC 2023] d='example.com'
[Mon Apr 17 08:23:21 PM UTC 2023] keyauthorization='ilNH3x8ybrSE8UFBxpevq72CtMhD13FEKA08bq8zo4w.FW1KGPj1HSHl9wu_d4wcIsBAj1CkQErlXT6vxu847yo'
[Mon Apr 17 08:23:21 PM UTC 2023] uri='https://10.6.18.10:14000/chalZ/iMsAmXHhZWscfLkCCDoCH9OyhdIeNOt3IWULgVHp4OE'
[Mon Apr 17 08:23:21 PM UTC 2023] _currentRoot='/var/www/html'
[Mon Apr 17 08:23:21 PM UTC 2023] wellknown_path='/var/www/html/.well-known/acme-challenge'
[Mon Apr 17 08:23:21 PM UTC 2023] writing token:ilNH3x8ybrSE8UFBxpevq72CtMhD13FEKA08bq8zo4w to /var/www/html/.well-known/acme-challenge/ilNH3x8ybrSE8UFBxpevq72CtMhD13FEKA08bq8zo4w
[Mon Apr 17 08:23:21 PM UTC 2023] Trigger domain validation.
[Mon Apr 17 08:23:21 PM UTC 2023] _t_url='https://10.6.18.10:14000/chalZ/iMsAmXHhZWscfLkCCDoCH9OyhdIeNOt3IWULgVHp4OE'
[Mon Apr 17 08:23:21 PM UTC 2023] _t_key_authz='ilNH3x8ybrSE8UFBxpevq72CtMhD13FEKA08bq8zo4w.FW1KGPj1HSHl9wu_d4wcIsBAj1CkQErlXT6vxu847yo'
[Mon Apr 17 08:23:21 PM UTC 2023] _t_vtype='http-01'
[Mon Apr 17 08:23:21 PM UTC 2023] url='https://10.6.18.10:14000/chalZ/iMsAmXHhZWscfLkCCDoCH9OyhdIeNOt3IWULgVHp4OE'
[Mon Apr 17 08:23:21 PM UTC 2023] payload='{}'
[Mon Apr 17 08:23:21 PM UTC 2023] Use cached jwk for file: /root/.acme.sh/ca/10.6.18.10/dir/account.key
[Mon Apr 17 08:23:21 PM UTC 2023] Use _CACHED_NONCE='puZ61M9QiAw_wGIld80Y3Q'
[Mon Apr 17 08:23:21 PM UTC 2023] nonce='puZ61M9QiAw_wGIld80Y3Q'
[Mon Apr 17 08:23:22 PM UTC 2023] POST
[Mon Apr 17 08:23:22 PM UTC 2023] _post_url='https://10.6.18.10:14000/chalZ/iMsAmXHhZWscfLkCCDoCH9OyhdIeNOt3IWULgVHp4OE'
[Mon Apr 17 08:23:22 PM UTC 2023] body='{"protected": "eyJub25jZSI6ICJwdVo2MU05UWlBd193R0lsZDgwWTNRIiwgInVybCI6ICJodHRwczovLzEwLjYuMTguMTA6MTQwMDAvY2hhbFovaU1zQW1YSGhaV3NjZkxrQ0NEb0NIOU95aGRJZU5PdDNJV1VMZ1ZIcDRPRSIsICJhbGciOiAiRVMyNTYiLCAia2lkIjogImh0dHBzOi8vMTAuNi4xOC4xMDoxNDAwMC9teS1hY2NvdW50LzRmYWE0Nzk1ODBjNWNmN2MifQ", "payload": "e30", "signature": "QVNbJ5VlzM5oWAbsPUYnh0FeQ4Gkr8NTj5QuXz6CTuiIFzQZP8CUCjeKQe7J0IQX8kgto1CHUhi2Ug_qaUOISA"}'
[Mon Apr 17 08:23:22 PM UTC 2023] _postContentType='application/jose+json'
[Mon Apr 17 08:23:22 PM UTC 2023] Http already initialized.
[Mon Apr 17 08:23:22 PM UTC 2023] _CURL='curl --silent --dump-header /root/.acme.sh/http.header  -L  --trace-ascii /tmp/tmp.42Wa6lwfxs  -g  --insecure  '
[Mon Apr 17 08:23:22 PM UTC 2023] _ret='0'
[Mon Apr 17 08:23:22 PM UTC 2023] responseHeaders='HTTP/2 200
cache-control: public, max-age=0, no-cache
content-type: application/json; charset=utf-8
link: <https://10.6.18.10:14000/dir>;rel="index"
link: <https://10.6.18.10:14000/authZ/9fGSeiH6Vf5QsZo_L9mOV_nMWyDGYEpxRPu3S0HanWA>;rel="up"
replay-nonce: zC_2ugmdse7Y7n9i_auiOg
content-length: 198
date: Mon, 17 Apr 2023 20:23:22 GMT
'
[Mon Apr 17 08:23:22 PM UTC 2023] code='200'
[Mon Apr 17 08:23:22 PM UTC 2023] original='{
   "type": "http-01",
   "url": "https://10.6.18.10:14000/chalZ/iMsAmXHhZWscfLkCCDoCH9OyhdIeNOt3IWULgVHp4OE",
   "token": "ilNH3x8ybrSE8UFBxpevq72CtMhD13FEKA08bq8zo4w",
   "status": "processing"
}'
[Mon Apr 17 08:23:22 PM UTC 2023] response='{"type":"http-01","url":"https://10.6.18.10:14000/chalZ/iMsAmXHhZWscfLkCCDoCH9OyhdIeNOt3IWULgVHp4OE","token":"ilNH3x8ybrSE8UFBxpevq72CtMhD13FEKA08bq8zo4w","status":"processing"}'
[Mon Apr 17 08:23:22 PM UTC 2023] trigger validation code: 200
[Mon Apr 17 08:23:22 PM UTC 2023] original='{"type":"http-01","url":"https://10.6.18.10:14000/chalZ/iMsAmXHhZWscfLkCCDoCH9OyhdIeNOt3IWULgVHp4OE","token":"ilNH3x8ybrSE8UFBxpevq72CtMhD13FEKA08bq8zo4w","status":"processing"}'
[Mon Apr 17 08:23:22 PM UTC 2023] response='{"type":"http-01","url":"https://10.6.18.10:14000/chalZ/iMsAmXHhZWscfLkCCDoCH9OyhdIeNOt3IWULgVHp4OE","token":"ilNH3x8ybrSE8UFBxpevq72CtMhD13FEKA08bq8zo4w","status":"processing"}'
[Mon Apr 17 08:23:22 PM UTC 2023] status='processing'
[Mon Apr 17 08:23:22 PM UTC 2023] Processing, The CA is processing your order, please just wait. (1/30)
[Mon Apr 17 08:23:22 PM UTC 2023] sleep 2 secs to verify again
[Mon Apr 17 08:23:25 PM UTC 2023] checking
[Mon Apr 17 08:23:25 PM UTC 2023] url='https://10.6.18.10:14000/chalZ/iMsAmXHhZWscfLkCCDoCH9OyhdIeNOt3IWULgVHp4OE'
[Mon Apr 17 08:23:25 PM UTC 2023] payload
[Mon Apr 17 08:23:25 PM UTC 2023] Use cached jwk for file: /root/.acme.sh/ca/10.6.18.10/dir/account.key
[Mon Apr 17 08:23:25 PM UTC 2023] Use _CACHED_NONCE='zC_2ugmdse7Y7n9i_auiOg'
[Mon Apr 17 08:23:25 PM UTC 2023] nonce='zC_2ugmdse7Y7n9i_auiOg'
[Mon Apr 17 08:23:25 PM UTC 2023] POST
[Mon Apr 17 08:23:25 PM UTC 2023] _post_url='https://10.6.18.10:14000/chalZ/iMsAmXHhZWscfLkCCDoCH9OyhdIeNOt3IWULgVHp4OE'
[Mon Apr 17 08:23:25 PM UTC 2023] body='{"protected": "eyJub25jZSI6ICJ6Q18ydWdtZHNlN1k3bjlpX2F1aU9nIiwgInVybCI6ICJodHRwczovLzEwLjYuMTguMTA6MTQwMDAvY2hhbFovaU1zQW1YSGhaV3NjZkxrQ0NEb0NIOU95aGRJZU5PdDNJV1VMZ1ZIcDRPRSIsICJhbGciOiAiRVMyNTYiLCAia2lkIjogImh0dHBzOi8vMTAuNi4xOC4xMDoxNDAwMC9teS1hY2NvdW50LzRmYWE0Nzk1ODBjNWNmN2MifQ", "payload": "", "signature": "v0xyRJvVBMLg1d73WuP5cCygDc7h2-F8RqKm6INy9DSVecI7iWf110iAaFuD60Rdrpwl7EEmrSoNflRyEZIs9w"}'
[Mon Apr 17 08:23:25 PM UTC 2023] _postContentType='application/jose+json'
[Mon Apr 17 08:23:25 PM UTC 2023] Http already initialized.
[Mon Apr 17 08:23:25 PM UTC 2023] _CURL='curl --silent --dump-header /root/.acme.sh/http.header  -L  --trace-ascii /tmp/tmp.42Wa6lwfxs  -g  --insecure  '
[Mon Apr 17 08:23:25 PM UTC 2023] _ret='0'
[Mon Apr 17 08:23:25 PM UTC 2023] responseHeaders='HTTP/2 200
cache-control: public, max-age=0, no-cache
content-type: application/json; charset=utf-8
link: <https://10.6.18.10:14000/dir>;rel="index"
replay-nonce: fOBAXs56hd6ebq04Em5Z0w
content-length: 523
date: Mon, 17 Apr 2023 20:23:25 GMT
'
[Mon Apr 17 08:23:25 PM UTC 2023] code='200'
[Mon Apr 17 08:23:25 PM UTC 2023] original='{
   "type": "http-01",
   "url": "https://10.6.18.10:14000/chalZ/iMsAmXHhZWscfLkCCDoCH9OyhdIeNOt3IWULgVHp4OE",
   "token": "ilNH3x8ybrSE8UFBxpevq72CtMhD13FEKA08bq8zo4w",
   "status": "invalid",
   "validated": "2023-04-17T20:23:22Z",
   "error": {
      "type": "urn:ietf:params:acme:error:unauthorized",
      "detail": "The key authorization file from the server did not match this challenge \"ilNH3x8ybrSE8UFBxpevq72CtMhD13FEKA08bq8zo4w.FW1KGPj1HSHl9wu_d4wcIsBAj1CkQErlXT6vxu847yo\" != \"\"",
      "status": 403
   }
}'
[Mon Apr 17 08:23:25 PM UTC 2023] response='{"type":"http-01","url":"https://10.6.18.10:14000/chalZ/iMsAmXHhZWscfLkCCDoCH9OyhdIeNOt3IWULgVHp4OE","token":"ilNH3x8ybrSE8UFBxpevq72CtMhD13FEKA08bq8zo4w","status":"invalid","validated":"2023-04-17T20:23:22Z","error":{"type":"urn:ietf:params:acme:error:unauthorized","detail":"The key authorization file from the server did not match this challenge \"ilNH3x8ybrSE8UFBxpevq72CtMhD13FEKA08bq8zo4w.FW1KGPj1HSHl9wu_d4wcIsBAj1CkQErlXT6vxu847yo\" != \"\"","status": 403}}'
[Mon Apr 17 08:23:26 PM UTC 2023] original='{"type":"http-01","url":"https://10.6.18.10:14000/chalZ/iMsAmXHhZWscfLkCCDoCH9OyhdIeNOt3IWULgVHp4OE","token":"ilNH3x8ybrSE8UFBxpevq72CtMhD13FEKA08bq8zo4w","status":"invalid","validated":"2023-04-17T20:23:22Z","error":{"type":"urn:ietf:params:acme:error:unauthorized","detail":"The key authorization file from the server did not match this challenge \"ilNH3x8ybrSE8UFBxpevq72CtMhD13FEKA08bq8zo4w.FW1KGPj1HSHl9wu_d4wcIsBAj1CkQErlXT6vxu847yo\" != \"\"","status": 403}}'
[Mon Apr 17 08:23:26 PM UTC 2023] response='{"type":"http-01","url":"https://10.6.18.10:14000/chalZ/iMsAmXHhZWscfLkCCDoCH9OyhdIeNOt3IWULgVHp4OE","token":"ilNH3x8ybrSE8UFBxpevq72CtMhD13FEKA08bq8zo4w","status":"invalid","validated":"2023-04-17T20:23:22Z","error":{"type":"urn:ietf:params:acme:error:unauthorized","detail":"The key authorization file from the server did not match this challenge \"ilNH3x8ybrSE8UFBxpevq72CtMhD13FEKA08bq8zo4w.FW1KGPj1HSHl9wu_d4wcIsBAj1CkQErlXT6vxu847yo\" != \"\"","status": 403}}'
[Mon Apr 17 08:23:26 PM UTC 2023] status='invalid'
[Mon Apr 17 08:23:26 PM UTC 2023] error='"error":{"type":"urn:ietf:params:acme:error:unauthorized","detail":"The key authorization file from the server did not match this challenge '
[Mon Apr 17 08:23:26 PM UTC 2023] errordetail='The key authorization file from the server did not match this challenge '
[Mon Apr 17 08:23:26 PM UTC 2023] example.com:Verify error:The key authorization file from the server did not match this challenge
[Mon Apr 17 08:23:26 PM UTC 2023] Debug: get token url.
[Mon Apr 17 08:23:26 PM UTC 2023] GET
[Mon Apr 17 08:23:26 PM UTC 2023] url='http://example.com/.well-known/acme-challenge/ilNH3x8ybrSE8UFBxpevq72CtMhD13FEKA08bq8zo4w'
[Mon Apr 17 08:23:26 PM UTC 2023] timeout=1
[Mon Apr 17 08:23:26 PM UTC 2023] Http already initialized.
[Mon Apr 17 08:23:26 PM UTC 2023] _CURL='curl --silent --dump-header /root/.acme.sh/http.header  -L  --trace-ascii /tmp/tmp.42Wa6lwfxs  -g  --insecure   --connect-timeout 1'
<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
htmlhead
title404 Not Foundtitle
head body
Not Found
The requested URL was not found on this server.
hr
addressApache/2.4.52 (Ubuntu) Server at example.com Port 80/address
body html
[Mon Apr 17 08:23:26 PM UTC 2023] ret='0'
[Mon Apr 17 08:23:26 PM UTC 2023] Debugging, skip removing: /var/www/html/.well-known/acme-challenge/ilNH3x8ybrSE8UFBxpevq72CtMhD13FEKA08bq8zo4w
[Mon Apr 17 08:23:26 PM UTC 2023] pid
[Mon Apr 17 08:23:26 PM UTC 2023] Using config home:/root/.acme.sh
[Mon Apr 17 08:23:26 PM UTC 2023] ACME_DIRECTORY='https://10.6.18.10:14000/dir'
[Mon Apr 17 08:23:26 PM UTC 2023] _ACME_SERVER_HOST='10.6.18.10'
[Mon Apr 17 08:23:26 PM UTC 2023] _ACME_SERVER_PATH='dir'
[Mon Apr 17 08:23:26 PM UTC 2023] httpdconfname='apache2.conf'
[Mon Apr 17 08:23:26 PM UTC 2023] httpdroot='/etc/apache2'
[Mon Apr 17 08:23:26 PM UTC 2023] httpdconf='/etc/apache2/apache2.conf'
[Mon Apr 17 08:23:26 PM UTC 2023] httpdconfname='apache2.conf'
[Mon Apr 17 08:23:26 PM UTC 2023] Restored: /etc/apache2/apache2.conf.
[Mon Apr 17 08:23:26 PM UTC 2023] Restored successfully.
[Mon Apr 17 08:23:26 PM UTC 2023] No need to restore nginx, skip.
[Mon Apr 17 08:23:26 PM UTC 2023] _clearupdns
[Mon Apr 17 08:23:26 PM UTC 2023] dns_entries
[Mon Apr 17 08:23:26 PM UTC 2023] skip dns.
[Mon Apr 17 08:23:26 PM UTC 2023] _on_issue_err
[Mon Apr 17 08:23:26 PM UTC 2023] Please check log file for more details: /root/.acme.sh/acme.sh.log
[Mon Apr 17 08:23:26 PM UTC 2023] _chk_vlist='example.com#ilNH3x8ybrSE8UFBxpevq72CtMhD13FEKA08bq8zo4w.FW1KGPj1HSHl9wu_d4wcIsBAj1CkQErlXT6vxu847yo#https://10.6.18.10:14000/chalZ/iMsAmXHhZWscfLkCCDoCH9OyhdIeNOt3IWULgVHp4OE#http-01#/var/www/html,'
[Mon Apr 17 08:23:26 PM UTC 2023] start to deactivate authz
[Mon Apr 17 08:23:26 PM UTC 2023] Trigger domain validation.
[Mon Apr 17 08:23:26 PM UTC 2023] _t_url='https://10.6.18.10:14000/chalZ/iMsAmXHhZWscfLkCCDoCH9OyhdIeNOt3IWULgVHp4OE'
[Mon Apr 17 08:23:26 PM UTC 2023] _t_key_authz='ilNH3x8ybrSE8UFBxpevq72CtMhD13FEKA08bq8zo4w.FW1KGPj1HSHl9wu_d4wcIsBAj1CkQErlXT6vxu847yo'
[Mon Apr 17 08:23:26 PM UTC 2023] _t_vtype
[Mon Apr 17 08:23:26 PM UTC 2023] url='https://10.6.18.10:14000/chalZ/iMsAmXHhZWscfLkCCDoCH9OyhdIeNOt3IWULgVHp4OE'
[Mon Apr 17 08:23:26 PM UTC 2023] payload='{}'
[Mon Apr 17 08:23:26 PM UTC 2023] Use cached jwk for file: /root/.acme.sh/ca/10.6.18.10/dir/account.key
[Mon Apr 17 08:23:26 PM UTC 2023] Use _CACHED_NONCE='fOBAXs56hd6ebq04Em5Z0w'
[Mon Apr 17 08:23:26 PM UTC 2023] nonce='fOBAXs56hd6ebq04Em5Z0w'
[Mon Apr 17 08:23:27 PM UTC 2023] POST
[Mon Apr 17 08:23:27 PM UTC 2023] _post_url='https://10.6.18.10:14000/chalZ/iMsAmXHhZWscfLkCCDoCH9OyhdIeNOt3IWULgVHp4OE'
[Mon Apr 17 08:23:27 PM UTC 2023] body='{"protected": "eyJub25jZSI6ICJmT0JBWHM1NmhkNmVicTA0RW01WjB3IiwgInVybCI6ICJodHRwczovLzEwLjYuMTguMTA6MTQwMDAvY2hhbFovaU1zQW1YSGhaV3NjZkxrQ0NEb0NIOU95aGRJZU5PdDNJV1VMZ1ZIcDRPRSIsICJhbGciOiAiRVMyNTYiLCAia2lkIjogImh0dHBzOi8vMTAuNi4xOC4xMDoxNDAwMC9teS1hY2NvdW50LzRmYWE0Nzk1ODBjNWNmN2MifQ", "payload": "e30", "signature": "LdKQcgPBQS6O9kmlA8ASRdymdbv6cY9p_8W7qLPsMmCj_22SEPDOZrQT9V7ONpCksZkSHUO1H3WJN3zLPyxScw"}'
[Mon Apr 17 08:23:27 PM UTC 2023] _postContentType='application/jose+json'
[Mon Apr 17 08:23:27 PM UTC 2023] Http already initialized.
[Mon Apr 17 08:23:27 PM UTC 2023] _CURL='curl --silent --dump-header /root/.acme.sh/http.header  -L  --trace-ascii /tmp/tmp.42Wa6lwfxs  -g  --insecure  '
[Mon Apr 17 08:23:27 PM UTC 2023] _ret='0'
[Mon Apr 17 08:23:27 PM UTC 2023] responseHeaders='HTTP/2 400
cache-control: public, max-age=0, no-cache
content-type: application/problem+json; charset=utf-8
link: <https://10.6.18.10:14000/dir>;rel="index"
replay-nonce: VaRAwvwaY3miPFbJSUU8OQ
content-length: 152
date: Mon, 17 Apr 2023 20:23:27 GMT
'
[Mon Apr 17 08:23:27 PM UTC 2023] code='400'
[Mon Apr 17 08:23:27 PM UTC 2023] original='{
   "type": "urn:ietf:params:acme:error:malformed",
   "detail": "Cannot update challenge with status invalid, only status pending",
   "status": 400
}'
[Mon Apr 17 08:23:27 PM UTC 2023] response='{
   "type": "urn:ietf:params:acme:error:malformed",
   "detail": "Cannot update challenge with status invalid, only status pending",
   "status": 400
}'
[Mon Apr 17 08:23:27 PM UTC 2023] Diagnosis versions:
openssl:openssl
OpenSSL 3.0.2 15 Mar 2022 (Library: OpenSSL 3.0.2 15 Mar 2022)
apache:
Server version: Apache/2.4.52 (Ubuntu)
Server built:   2023-03-08T17:32:01
Server's Module Magic Number: 20120211:121
Server loaded:  APR 1.7.0, APR-UTIL 1.6.1
Compiled using: APR 1.7.0, APR-UTIL 1.6.1
Architecture:   64-bit
Server MPM:     event
  threaded:     yes (fixed thread count)
    forked:     yes (variable process count)
Server compiled with....
 -D APR_HAS_SENDFILE
 -D APR_HAS_MMAP
 -D APR_HAVE_IPV6 (IPv4-mapped addresses enabled)
 -D APR_USE_PROC_PTHREAD_SERIALIZE
 -D APR_USE_PTHREAD_SERIALIZE
 -D SINGLE_LISTEN_UNSERIALIZED_ACCEPT
 -D APR_HAS_OTHER_CHILD
 -D AP_HAVE_RELIABLE_PIPED_LOGS
 -D DYNAMIC_MODULE_LIMIT=256
 -D HTTPD_ROOT="/etc/apache2"
 -D SUEXEC_BIN="/usr/lib/apache2/suexec"
 -D DEFAULT_PIDLOG="/var/run/apache2.pid"
 -D DEFAULT_SCOREBOARD="logs/apache_runtime_status"
 -D DEFAULT_ERRORLOG="logs/error_log"
 -D AP_TYPES_CONFIG_FILE="mime.types"
 -D SERVER_CONFIG_FILE="apache2.conf"
nginx:
nginx doesn't exist.
socat:
socat by Gerhard Rieger and contributors - see www.dest-unreach.org
socat version 1.7.4.1 on Mar 25 2022 09:51:32
   running on Linux version #76-Ubuntu SMP Fri Mar 17 17:19:29 UTC 2023, release 5.15.0-69-generic, machine x86_64
features:
  #define WITH_STDIO 1
  #define WITH_FDNUM 1
  #define WITH_FILE 1
  #define WITH_CREAT 1
  #define WITH_GOPEN 1
  #define WITH_TERMIOS 1
  #define WITH_PIPE 1
  #define WITH_UNIX 1
  #define WITH_ABSTRACT_UNIXSOCKET 1
  #define WITH_IP4 1
  #define WITH_IP6 1
  #define WITH_RAWIP 1
  #define WITH_GENERICSOCKET 1
  #define WITH_INTERFACE 1
  #define WITH_TCP 1
  #define WITH_UDP 1
  #define WITH_SCTP 1
  #define WITH_LISTEN 1
  #define WITH_SOCKS4 1
  #define WITH_SOCKS4A 1
  #define WITH_VSOCK 1
  #define WITH_PROXY 1
  #define WITH_SYSTEM 1
  #define WITH_EXEC 1
  #undef WITH_READLINE
  #define WITH_TUN 1
  #define WITH_PTY 1
  #define WITH_OPENSSL 1
  #undef WITH_FIPS
  #define WITH_LIBWRAP 1
  #define WITH_SYCLS 1
  #define WITH_FILAN 1
  #define WITH_RETRY 1
  #define WITH_MSGLEVEL 0 /*debug*/

**Comment by wlmarion** [comment]
Debug logs already posted. Version is also up to date.

**Comment by neilpang** [comment]
Closing as stale: no activity for over 18 months. If this still applies on the current release, please open a new issue with the `--debug 2` log.
